### ISSUE TITLE
Include CSS, minified in NPM build

### ISSUE THREAD

**Issue by tony** [ava-cassiopeia/simple-switch#22]
Thank you for the project!  I have a clarification on something:

in re: [README as of v0.8.0](https://github.com/ava-cassiopeia/simple-switch/blob/v0.8.0/readme.md#installing-the-css)

> The minified Javascript file is available on the [releases page](https://github.com/aeolingamenfel/simple-switch/releases), within the release .zip file, under js/.

> You can install the CSS by downloading the compiled CSS file from the [releases page](https://github.com/aeolingamenfel/simple-switch/releases), under css/.

Is there any earlier conversation or context behind these behind in GitHub releases, @ava-cassiopeia?

Are you open to including this is the NPM builds? What do you think works best?

**Comment by tony** [comment]
P.S. There's a lot of action going on in NPM. If for some reason newer NPM versions can download GitHub releases (even then, I still think having it in the npm build is convenient), or I'm missing something else obvious - please let me know!

**Comment by ava-cassiopeia** [comment]
> Is there any earlier conversation or context behind these behind in GitHub releases

Not really, I just also provide the base files in case someone doesn't use NPM (for some reason?) but still wants to use this without having to clone-install-build to get the files.

>Are you open to including this is the NPM builds?

Sure! I don't see why not &mdash; if I put it in the right folder within the release it shouldn't interfere with normal NPM imports I believe. I'll issue a new release on NPM sometime today (ideally) and will update this issue when that happens.

**Comment by ava-cassiopeia** [comment]
@tony as-of `a-simple-switch@0.8.1` on NPM, I've included all of the files that are normally only packed into GitHub releases. Please let me know if this is working as intended for your needs!

**Comment by tony** [comment]
@ava-cassiopeia I don't see 0.8.1 yet. Did you publish it?

**Comment by ava-cassiopeia** [comment]
My apologies, I mistyped a command and didn't upload it originally. 0.8.1 and 0.9.0 are both now available on NPM.

> I can make a repo / package for [the lit element wrapper] if you'd like

Honestly most of the point of this was to avoid framework-specific code. Personally if I were to make a version of this for other frameworks, I'd probably write it from the ground up in that framework to take advantage of framework-specific wrapping and bindings.

That being said, I don't really have any plans to create a bunch of different versions of this for various frameworks, so if you want to publish a lit html variant of this that's fine with me &mdash; I will happily link to it from this component's readme!

One other thing to note: to my knowledge there is no Material implementation for lit html, but there is for React, so if you want the React version of the Material switch, I'd recommend that!

**Comment by ava-cassiopeia** [comment]
Closing since this has been open for awhile &mdash; feel free to reopen if desired!
